### PR TITLE
Update `ucx-py` version

### PR DESCRIPTION
**Pull request by ajschmidt8**
This PR updates the `ucx-py` version for `branch-22.06`.

### CODE REVIEW

File: ci/release/update-version.sh [hunk 1]
```diff
@@ -39,3 +39,4 @@ for FILE in conda/environments/*.yml; do
 done
 
 sed_runner "s/export UCX_PY_VERSION=.*/export UCX_PY_VERSION='${NEXT_UCX_PY_VERSION}'/g" ci/gpu/build.sh
+sed_runner "s/export UCX_PY_VERSION=.*/export UCX_PY_VERSION='${NEXT_UCX_PY_VERSION}'/g" ci/cpu/build.sh
```
Comment (dantegd) [inline ci/release/update-version.sh:42]: isn't this a duplicate of the line?

Comment (ajschmidt8) [reply ci/release/update-version.sh:42]: almost. one is `cpu/build.sh` the other is `gpu/build.sh`

Comment (jakirkham) [reply ci/release/update-version.sh:42]: This one is `cpu` and the previous one is `gpu`

**Comment by ajschmidt8** [comment]
@gpucibot merge

**Review by dantegd — APPROVED** [review]

**Review by msadang — APPROVED** [review]
LGTM

**Review by cjnolet — APPROVED** [review]
LGTM

**Pull request merged**